### ISSUE TITLE
Broken linking since rustc 1.17.0-beta.2/cargo 0.18.0-beta

### ISSUE THREAD

**Issue by kud1ing** [kud1ing/rucaja#12]
```
error while loading shared libraries:
libjvm.so: cannot open shared object file: No such file or directory
```

**Comment by treyzania** [comment]
Perhaps you should change your approach to the dynamic linking: find the dynamic libraries at runtime instead of locking it in at build time.

The crate `rust-libloading` looks pretty good, but you'd have to change your approach to interacting with the JVM and write a decent amount of wrapper code.

**Comment by kud1ing** [comment]
Fixed by #15
